doc/source/serve/production-guide/kubernetes.md

```diff
@@ -66,14 +66,14 @@ Learn more about how to configure KubeRay clusters [here](kuberay-config).
 :::
 
 ```console
-$ curl -o ray_v1alpha1_rayservice.yaml https://raw.githubusercontent.com/ray-project/kuberay/release-0.5/ray-operator/config/samples/ray_v1alpha1_rayservice.yaml
+$ curl -o ray-service.text-ml.yaml https://raw.githubusercontent.com/ray-project/kuberay/master/ray-operator/config/samples/ray-service.text-ml.yaml
```

Comment:
I think it might be better to pin this to a specific release or commit. Also note https://raw.githubusercontent.com/ray-project/kuberay/master/ray-operator/config/samples/ray-service.text-ml.yaml is currently throwing 404

Reply:
Will do! This is a placeholder right now because https://github.com/ray-project/kuberay/pull/1402 hasn't been merged yet, I will update this link to the commit once it's merged.